__tests__/utils.test.js

```diff
@@ -244,6 +244,72 @@ describe("Northcoders News API", () => {
         });
     });
   });
+
+
+
+
+
+
+  describe('CORE: PATCH /api/articles/:article_id', () => {
+    it('200 - Respond with the votes column being added with article_id 1', () => {
+      const voteObj = {
+        inc_votes : 10
+      }
+      return request(app).patch('/api/articles/1').send(voteObj).expect(200).then(({ body }) => {
+        expect(body.article.votes).toBe(110)
+        expect(body.article.article_id).toBe(1)
+        expect(body.article.title).toBe('Living in the shadow of a great man')
+        expect(body.article.author).toBe('butter_bridge')
+        expect(body.article.body).toBe('I find this existence challenging')
+        expect(body.article.created_at).toBe('2020-07-09T20:11:00.000Z')
+      })
+    })
+
+    it('200 - Respond with the votes column being subtracted with article_id 2', () => {
+      const voteObj = {
+        inc_votes : -10
+      }
+      return request(app).patch('/api/articles/2').send(voteObj).expect(200).then(({ body }) => {
+        expect(body.article.votes).toBe(voteObj.inc_votes)
+      })
+    })
+
+    it('400 - respond with a bad request if the user forgets to enter a vote count', () => {
+      const voteObj = {
+        inc_votes : ''
+      }
+      return request(app).patch('/api/articles/1').send(voteObj).expect(400).then(( response ) => {
+        expect(response.body.msg).toBe('Bad request')
+        expect(response.badRequest).toBe(true)
+        expect(response.statusCode).toBe(400)
```

Comment:
You are already expecting a 400 at this point, so your expects on line 283 and 284 are rather redundant.